src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -2274,12 +2601,18 @@ read_comm(ssize_t (*next_read)(int fd, void *buf, size_t size), int fd, void *bu
 	if (!hook_enabled)
 		return next_read(fd, buf, size);
 
+	if (is_bash && fd <= 2 && compatible_mode)
```

Comment:
I think technically if I've understood this right is_bash will only be set for compatible mode so this could be reduced?
```suggestion
	if (is_bash && fd <= 2)
```

Reply:
You are right. Fixed as you suggested. Thank you!